src/core/arm/dynarmic/arm_dynarmic_32.cpp

```diff
@@ -450,7 +450,7 @@ std::vector<ARM_Interface::BacktraceEntry> ARM_Dynarmic_32::GetBacktrace(Core::S
     // Frame records are two words long:
     // fp+0 : pointer to previous frame record
     // fp+4 : value of lr for frame
-    while (true) {
+    for (size_t i = 0; i < 100; i++) {
```

Comment:
Extremely nitpicky on my part, would love to see a nice round 2^8
```suggestion
    for (size_t i = 0; i < 256; i++) {
```